scripts/linters/general_purpose_linter.py

```diff
@@ -132,6 +133,16 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
+    {
+        'regexp': re.compile(r'bypassSecurityTrust'),
```

Comment:
Be stricter. Go with "bypass" and say something like "The use of 'bypass' is not allowed, particularly with regards to bypassSecurityTrustedUrl() and similar functions in Angular."

(Chances are that, if "bypass" is used for something, anywhere, we'd want to know.)

Reply:
Done!